### ISSUE TITLE
Lecture this week? 

### ISSUE THREAD

**Issue by jnazaren** [dilevin/computer-graphics-csc418#17]
I'm seeing that there's no lecture on the schedule for week 7. Does that mean theres no lecture tomorrow (2/25)?

**Comment by dilevin** [comment]
Correct. I just sent an announcement about this through Quercus but I'll post it here as well:

"Hi Everyone,

No official lectures Tuesday or Wednesday :)

I will use that time to do 3 hour office/tutorial hours (3-6pm) on both days.

3-5pm find me in GB 244

5-6pm find me in my office.

-Dave "

**Comment by dilevin** [comment]
Hi Everyone

Unfortunately I need to cancel the 3-6pm tutorial/office hours today due to a personal matter.I will still do a 3-6pm  tutorial/office hours  tomorrow (3-5pm in GB221, 5-6pm) in my office.

For pressing matters please reach out via GitHub issues or email (whichever is appropriate).

Apologies,
-Dave
